File: C/Factorial.c
#include<stdio.h>
int fact(int n){
	if(n==1) return n;
	else return n*fact(n-1);
}
int main()
{
	int n;
	printf("Enter a number to find factoial: ");
	scanf("%d",&n);
	printf("The factorial of the number %d is %d",n,fact(n));
}

File: C#/factorial.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace FactorialExample
{
class Program
{
static void Main()
{
Console.WriteLine("Enter the number");
int n = Convert.ToInt32(Console.ReadLine());
long fact = Fact(n);
Console.WriteLine("factorial is {1}", n, fact);
Console.ReadKey();
}
private static long Fact(int n)
{
if (n == 0)
{
return 1;
}
return n * Fact(n - 1);
}
}
}